src/components/body/body.component.ts

```diff
@@ -682,7 +682,8 @@ export class DataTableBodyComponent implements OnInit, OnDestroy {
    * Gets the row index given a row
    */
   getRowIndex(row: any): number {
-    return this.rowIndexes.get(row) || 0;
+    // return this.rowIndexes.get(row) || 0;
```

Comment:
@Jotakuun The `indexOf` array function is much slower than the Map `get` function. Are you having issues with it returning the wrong value?

Reply:
Ok, so it looks like the getRowIndex() was changed in last release and this piece of code wasn't working for rowGroups because rowIndexes is not returning any index for rowGroups. I couldn't figure it out why. Any suggestion?

Reply:
hmm, not sure. Didn't really get a chance to look over the row groups PR that much. I can tell you that `rowIndexes` should have a list of indexes per row, same as the `rows` array being searched manually, but much slower. If `rowIndexes` isn't producing correct results, then it's not being updated correctly.

Reply:
Grouped row indexes are in a different collection.

Reply:
And where is it? The ```rowIndexes``` are not updating in the ```updateRow()``` method when there's groupedRows.

Reply:
Can we treat this is a separate issue? I find value in moving the css out of the `material.scss` as it's causing issues.

Reply:
i removed it and released thisx morning.

Reply:
I agree, this is completely another issue. I update the PR again with just the ```material.scss``` and we treat this in another pull request.